deployment/start_and_deploy_to_server.sh

```diff
@@ -48,19 +48,18 @@ scp -r "$construction_dir" ubuntu@"$target_server_url":"$location"
 ssh ubuntu@"$target_server_url" "mv $keycloak_backup_dir/*-users-*.json $keycloak_user_dir"
 
 echo "Set up Keycloak from scratch."
-ssh ubuntu@"$target_server_url" "export KEYCLOAK_FRONTEND_URL=\"$KEYCLOAK_FRONTEND_URL\";
```

Comment:
You should not remove the KEYCLOAK_FRONTEND_URL as long as it occurs in the realm files. It occurs in datalandsecurity-realm.json.

Reply:
I did not remove it, I just changed the place where it is declared:
Before my changes, it was declared here, because only the keycloak initialization process needed this variable.

Now after implementing the email function, besides the keycloak initialization process, also the running keycloak service itself needs this variable.
And that's why I declared it in the .env file which is build during the "buildDirectories()" function in our bash CD script, because that way it is accessible for all docker-compose services.
Since it is no "security-critical" info like a password or key or something like that, this should be ok.